transformers/PDFTransformer.py

```diff
@@ -81,6 +84,9 @@ def get_elements(self):
 
         return elements
 
+    def get_localised_date(self, timezone='Europe/London'):
```

Comment:
I think it would be cleaner to explicitly pass `submitted_at` to `get_localised_date` as the method name implies it will get the localised version of any date, not just the `submitted_at` date.